etc/init/rush-standalone.conf

```diff
@@ -0,0 +1,10 @@
+# auto start rush service if rush is installed standalone
+
+description "rush-standalone"
+
+start on filesystem or runlevel [2345]
+stop on shutdown
+
+script
+  [ ! -f /etc/init/spinnaker.conf ] && { start rush; }
```

Comment:
I think re-writing the top level spinnaker script is a good idea.  I think one thing to also consider is that instead of assuming you would install a service you instead always install spinnaker and disable services you do not need if it's a stand alone scenario.

$HOME is easy to swap in to replace hard coded HOME